.storybook/main.js

```diff
@@ -35,6 +36,7 @@ module.exports = {
     "@storybook/addon-links",
     "@storybook/addon-a11y",
     "@storybook/addon-interactions",
+    "storybook-addon-pseudo-states",
```

Comment:
using `userEvent.hover` from `@storybook/addon-interactions` doesn't simulate the hover state, so we need this addon.